godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.java

```diff
@@ -203,11 +205,6 @@ public Dictionary confirmPriceChange(String sku) {
 		}
 
 		SkuDetails skuDetails = skuDetailsCache.get(sku);
-
-		PriceChangeFlowParams priceChangeFlowParams = 
-			PriceChangeFlowParams.newBuilder().setSkuDetails(skuDetails).build();
-
-		billingClient.launchPriceChangeConfirmationFlow(getActivity(), priceChangeFlowParams, this);
```

Comment:
`PriceChangeFlowParams` is [deprecated](https://developer.android.com/reference/com/android/billingclient/api/PriceChangeFlowParams)